docs/docs/_available-tags.md

```diff
@@ -0,0 +1,21 @@
+# Available tags
+
+The following is a list of available tags that can be used when contributing a sample. Click on the links to view the samples associated with the tags.
+
+Tag | Description
+-- | --
+[AI](./tags#AI) | Content dealing with AI related subjects. 
```

Comment:
Not all descriptions are equally valuable. Eg. _AI - Content dealing with AI-related subjects_ is pretty self-explanatory, and doesn't add any value. On the other hand _Migration - Content about exporting, importing, syncing or moving data_ is better because it explains what migration is. Given that for all tags we're basically repeating the tag, we should revisit the stance of adding descriptions and if there's nothing to clarify, we should consider removing them altogether.

Reply:
Ok, I agree. a lot of it is self explanatory, so I removed them. @garrytrinder